src/main/java/projet/projetstage02/service/GestionnaireService.java

```diff
@@ -409,4 +408,139 @@ public ContractsDTO getContractsToEvaluateMillieuStage() {
+
+
+        evaluationParagraph.put("Les taches confiées sont celles annoncées dans l'entente de stage",
+                fieldToText(evaluationMillieuStage.getTachesAnnonces()));
+        evaluationParagraph.put("Des mesures d'accueil facilitent l'intégration du nouveau stagiaire",
+                fieldToText(evaluationMillieuStage.getIntegration()));
+        evaluationParagraph.put("Le temps réel consacré à l'encadrement du stagiaire est suffisant",
+                fieldToText(evaluationMillieuStage.getTempsReelConsacre()));
+        evaluationParagraph.put("Nombre d'heures pour le premier mois",
+                evaluationMillieuStage.getHeureTotalPremierMois() + " heures");
+        evaluationParagraph.put("Nombre d'heures pour le deuxième mois",
+                evaluationMillieuStage.getHeureTotalDeuxiemeMois() + " heures");
+        evaluationParagraph.put("Nombre d'heures pour le troisième mois",
+                evaluationMillieuStage.getHeureTotalTroisiemeMois() + " heures");
+        evaluationParagraph.put("L'environnement de travail respecte les normes de sécurité",
+                fieldToText(evaluationMillieuStage.getEnvironementTravail()));
+        evaluationParagraph.put("Le climat de travail est agréable",
+                fieldToText(evaluationMillieuStage.getClimatTravail()));
+        evaluationParagraph.put("Le millieu de stage est accessible par transport en commun",
+                fieldToText(evaluationMillieuStage.getMilieuDeStage()));
+        evaluationParagraph.put("La communication avec le superviseur est efficace",
+                fieldToText(evaluationMillieuStage.getCommunicationAvecSuperviser()));
+        evaluationParagraph.put("L'équipement de travail est adéquat",
+                fieldToText(evaluationMillieuStage.getEquipementFourni()));
+        evaluationParagraph.put("Le volume de travail est raisonnable",
+                fieldToText(evaluationMillieuStage.getVolumeDeTravail()));
+
+        commentaires.put("", evaluationMillieuStage.getCommentaires());
+
+        signPara.put("Signé le", evaluationMillieuStage.getDateSignature());
+        map.put("Information sur la compagnie", companyInfo);
+        map.put("Information sur l'étudiant", studentInfo);
+        map.put("Information sur l'offre de stage", offerInfo);
+        map.put("Évaluation", evaluationParagraph);
+        if (evaluationMillieuStage.getCommentaires() != null && !evaluationMillieuStage.getCommentaires().isEmpty()) {
+            map.put("Commentaires", commentaires);
+        }
+        map.put("Signature", signPara);
+        map.put("_signature_", getSignatureGestionnaire(evaluationMillieuStage));
+        return map;
+    }
+
+    private Map<String, String> getSignatureGestionnaire(EvaluationMillieuStage evaluationMillieuStage) throws EmptySignatureException {
+        String signature = evaluationMillieuStage.getSignature();
+        String[] split = signature.split(",");
```

Comment:
Why do you use a split for a signature?

Reply:
There is some metadata in the begining of the string before the actual data